src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,122 @@
+# Setting Up Third-Party OAuth Integration
+
+## Overview
+
+GameFabric supports Third-Party OAuth Integration with every OAuth-compatible Identiy Provider, allowing you to reuse your existing authentication flows, without giving GameFabric access to your sensitive authentication data.
+
+## Prerequisites
+
+Before setting up a Third-Party OIDC Provider, ensure the following conditions are met:
+
+- You have access to an active GameFabric installation with access through the default user.
+- You have an OIDC ([OpenID Connect](https://openid.net/developers/how-connect-works/)) provider, including a **Client ID** and **Client Secret**.
+
+## Setting up a new OIDC Provider
+
+### Step 1: Add a New Provider
+
+1. Navigate to the OIDC Providers page (From Sidebar -> Access Management -> OIDC Providers).
+2. Click **Add Provider**.
+
+![Add Provider Screenshot](images/authentication/create-oidc-provider-main-screenshot.png)
+
+3. Enter a desired **ID** and **Display Name** for your new provider.
+
+![Provider Details Screenshot](images/authentication/create-oidc-provider-firststep-screenshot.png)
+
+### Step 2: Configure Issuer and Credentials
+
+1. Specify the **Issuer URL** of your OIDC provider.
+2. Enter the **Client ID** and **Client Secret** issued by the provider.
+
+![Issuer Configuration Screenshot](images/authentication/create-oidc-provider-secondstep-screenshot.png)
+
+### Step 3: Advanced Details
+
+#### Define Claim Configurations
+
+1. Configure the claims to request from the OIDC provider using the **Scopes** text input.
+
+![Scopes Screenshot](images/authentication/create-oidc-provider-thirdstep-screenshot.png)
+
+ Common scopes include:
+
+| Scope         | Description                                                               |
+|---------------|---------------------------------------------------------------------------|
+| openid        | Required for OpenID Connect; requests and ID token                        |
+| email         | Requests the user's email address                                         |
+| profile       | Requests additional user profile information, such as name and picture    |
+| groups        | Requests group membership claims (if supported by the identity provider)  |
+| custom scopes | Applications or APIs can define their own scopes for specific permissions |
+
+
+2. Define the following mappings:
+   - **UserID Key**: Maps the OIDC provider’s claim to the **User ID** in GameFabric (default: `sub`).
+   - **UserName Key**: Maps the OIDC provider’s claim to the **UserName** in GameFabric (default: `name`).
```

Comment:
This does not read well in this form. A sentence telling them that the software needs these values to be able to determine the user ID and name would be better IMO.